src/game/option/option_compass.c

```diff
@@ -88,10 +88,11 @@ static void Option_CompassInitText(void)
     m_Text[TEXT_TIME] = Text_Create(0, y, " ");
     y += row_height;
 
-    Text_AddBackground(m_Text[TEXT_TITLE_BORDER], row_width, y - top_y, 0, 0);
-    Text_AddOutline(m_Text[TEXT_TITLE_BORDER], 1);
-    Text_AddBackground(m_Text[TEXT_TITLE], row_width - 4, 0, 0, 0);
-    Text_AddOutline(m_Text[TEXT_TITLE], 1);
+    Text_AddBackground(
+        m_Text[TEXT_TITLE_BORDER], row_width, y - top_y, 0, 0, TS_BACKGROUND);
+    Text_AddOutline(m_Text[TEXT_TITLE_BORDER], true, TS_BACKGROUND);
+    Text_AddBackground(m_Text[TEXT_TITLE], row_width - 4, 0, 0, 0, TS_HEADING);
+    Text_AddOutline(m_Text[TEXT_TITLE], true, TS_HEADING);
```

Comment:
Consider deleting `Text_AddOutline` and `Text_AddBackground` altogether and instead adding `Text_SetStyle(TEXTSTRING *, TEXT_STYLE)`. Then the renderer knows what to do.

Reply:
Text_AddBackground needs the x, y, width, and height. So a `Text_SetStyle` would need to take this in. Not sure we want to do that?